File: Source/Fantasy.Jobs/Resources/TaskRuntimeScheduleSettings.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.Configuration;
using System.Xml.Serialization;

namespace Fantasy.Jobs.Resources
{
    public class TaskRuntimeScheduleSettings : RuntimeScheduleSetting
    {

        private List<TaskRuntimeScheduleSetting> _tasks = new List<TaskRuntimeScheduleSetting>();
         [XmlArray("tasks"),
        XmlArrayItem(ElementName = "task", Type = typeof(TaskRuntimeScheduleSetting))]
        public List<TaskRuntimeScheduleSetting> Tasks
        {
            get { return _tasks; }
        }
    }


    public class TaskRuntimeScheduleSetting : RuntimeScheduleSetting
    {
        [XmlAttribute("name")]
        public string Name { get; set; }

        [XmlAttribute("namespace")]
        public string Namespace { get; set; }

       
    }
}

File: Fantasy.Jobs/src/fantasy/jobs/resources/TaskRuntimeScheduleSettings.java
package fantasy.jobs.resources;

import Fantasy.Configuration.*;

public class TaskRuntimeScheduleSettings extends RuntimeScheduleSetting
{

	private java.util.ArrayList<TaskRuntimeScheduleSetting> _tasks = new java.util.ArrayList<TaskRuntimeScheduleSetting>();
//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	 //[XmlArray("tasks"), XmlArrayItem(ElementName = "task", java.lang.Class = typeof(TaskRuntimeScheduleSetting))]
	 public final java.util.ArrayList<TaskRuntimeScheduleSetting> getTasks()
	 {
		 return _tasks;
	 }
}